admin_panel/src/commonMain/kotlin/net/thechance/mena/admin_panel/data/remote/api_service/DepositApiService.kt

```diff
@@ -0,0 +1,17 @@
+package net.thechance.mena.admin_panel.data.remote.api_service
+
+import de.jensklingenberg.ktorfit.Response
+import de.jensklingenberg.ktorfit.http.Body
+import de.jensklingenberg.ktorfit.http.POST
+import net.thechance.mena.admin_panel.data.remote.dto.deposit.DepositRequestDto
+
+interface DepositApiService {
+    @POST(DEPOSIT_END_POINT)
```

Comment:
I prefer to make this function in a single line like this:

```suggestion
    @POST(DEPOSIT_END_POINT)
    suspend fun deposit(@Body deposit: DepositRequestDto): Response<Unit>
```

Reply:
I will not change it for consistency

Reply:
@gehadepada I can't understand consistency in what

Reply:
other files